{{cookiecutter.project_name}}/Makefile

```diff
@@ -171,26 +171,38 @@ test: test-$(TEST_RUNNER)
 .PHONY: tests
 tests: tests-$(TEST_RUNNER)
 
+.PHONY: read-coverage
+read-coverage:
+	$(OPEN) .coverage-html/index.html
+
 # nosetest commands
 
 .PHONY: test-nose
 test-nose: depends-ci
+	$(COVERAGE) erase; rm -rf .coverage-html
```

Comment:
Thoughts on having a `.clean-cov` target that takes care of this and using it as a dependency of these 4 tasks that erase coverage before running?